### DISCUSSION TITLE
Toggle to disable switching layers via object selection

### DISCUSSION THREAD

**Discussion by jsllllc** [keiwando/toonsquid#298] [Ideas]
I would appreciate a settings option to disable the automatic layer switching when accidentally selecting an object on a different layer. 

 While I can see how it could be time-saving for certain workflows, I often make errors by accidentally moving / adjusting a layer I didn't intend to switch to.  Oftentimes I do not notice these errors when they happen. The only workaround I have found is to lock all other layers every time I switch.

Thank you for your efforts.

**Comment by keiwando** [comment]
So an option to disable layer selection on the canvas and only allow layer selection via the layer lists, right?

**Reply by jsllllc** [reply]
Yes, that is what I was thinking! Thanks for your response.
